packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -1405,6 +1406,22 @@ const useEcharts = (
         explore,
     ]);
 
+    const tooltip = useMemo<EChartsOption['tooltip']>(
+        () => ({
+            show: true,
+            confine: true,
+            trigger: 'axis',
+            enterable: true,
```

Comment:
allows tooltip to be enterable, to then enable scrolling with the css set in `extraCssText`